### PR TITLE
feat: add pages to guide through the reuse declaration procedure

### LINKED ISSUE #799: Clarify to PAs how to declare they're using the software
Add a button to the software pages that leads to a page clarifying to PAs how to declare they're using that particular software.

### PR DESCRIPTION
**Pull request by bfabio**
Fix #799.

### CODE REVIEW

File: it/come-dichiaro-riuso.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+title: Dichiarazione di presa a riuso di software
```
Comment (libremente) [inline it/come-dichiaro-riuso.md:2]: I would call the file `dichiarazione.md` for consistency with the other files in /riuso

**Comment by bfabio** [comment]
Used `localStorage` to pass the repo URL to the new page, so we're not vulnerable to abuse (phishing, social engineering) by direct manipulation of the query string.

The state doesn't persist after a refresh of the page (ie, a refresh will show the whole page and the link to the repo will disappear), but I think it's a good tradeoff.

File: it/riuso/dichiarazione.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+     all’interno del file `publiccode.yml` attraverso lo strumento di ticketing (_issue_)
+   * inserimento del _nome_ e _codiceIPA_ del proprio ente direttamente
+     all’interno del campo `usedBy` del `publiccode.yml` tramite una _Pull Request_.
+
+In questo modo il manutentore del software potrà concludere le operazioni in
+autonomia e il nome del proprio Ente verrà visualizzato all’interno del catalogo.
+
+<div class="show-if-not-repo">
+  <h3>
+    Se il software è presente nel catalogo di Developers Italia.
```
Comment (libremente) [inline it/riuso/dichiarazione.md:59]:
```suggestion
    Se il software non è presente nel catalogo di Developers Italia
```

File: it/riuso/dichiarazione.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+Questa segnalazione permette all’Ente cedente di avere contezza di chi stia
+attivamente utilizzando un dato software e **facilitarne in questo modo
+l'interazione.**
+
+<div class="show-if-not-repo">
+  Le operazioni necessarie per effettuare la dichiarazione
+  variano in base all’esistenza o meno del software all’interno del catalogo.
+
+  <h3>
+    Se il software non è presente nel catalogo di Developers Italia
```
Comment (libremente) [inline it/riuso/dichiarazione.md:34]:
```suggestion
    Se il software è presente nel catalogo di Developers Italia
```

**Review by libremente — CHANGES_REQUESTED** [review]
I like it, however I believe we should work a bit more on the "visual" style. What about using some BSI components? Somthing like buttons, callouts, collapse or maybe divide the different actions in three horizontal sections in the page?

File: it/riuso/dichiarazione.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+
+Questa segnalazione permette all’Ente cedente di avere contezza di chi stia
+attivamente utilizzando un dato software e **facilitarne in questo modo
+l'interazione.**
+
+<div class="show-if-not-repo">
+  Le operazioni necessarie per effettuare la dichiarazione
+  variano in base all’esistenza o meno del software all’interno del catalogo.
+
+  <h3>
```
Comment (libremente) [inline it/riuso/dichiarazione.md:33]:
```suggestion
  <h2>
```

File: it/riuso/dichiarazione.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+    Se il software è presente nel catalogo di Developers Italia
+  </h3>
```
Comment (libremente) [inline it/riuso/dichiarazione.md:35]:
```suggestion
  </h2>
```

Comment (libremente) [reply it/riuso/dichiarazione.md:35]: Can we try with h2? I don't particularly like the h3 style applied in the page

File: it/riuso/dichiarazione.md [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+      </div>
+    </div>
+  </div>
+</div>
+
+In questo modo il manutentore del software potrà concludere le operazioni in
+autonomia e il nome del proprio Ente verrà visualizzato all’interno del catalogo.
+
+<div class="show-if-not-repo">
+  <h3>
```
Comment (libremente) [inline it/riuso/dichiarazione.md:103]: Same as above

File: it/riuso/dichiarazione.md [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+  <p>
+    Nel caso in cui il software preso a riuso non dovesse ancora essere
+    entrato all’interno del catalogo per questioni tecniche oppure dovesse essere
+    in procinto di terminare le operazioni di <a href="/it/riuso/pubblicazione">pubblicazione</a>
+    <strong>è comunque possibile portare a termine questa dichiarazione.</strong>
+  </p>
+</div>
+
+**Tramite gli strumenti di comunicazione** messi a disposizione da Developers Italia,
+quali ad esempio Slack oppure il Forum è sempre possibile avviare una
```
Comment (libremente) [inline it/riuso/dichiarazione.md:126]:
```suggestion
quali ad esempio Slack oppure il Forum, è sempre possibile avviare una
```

File: it/riuso/dichiarazione.md [hunk 8]
```diff
@@ -0,0 +1,161 @@
...
+discussione e il **Team di Developers Italia interverrà per aiutare a completare
```
Comment (libremente) [inline it/riuso/dichiarazione.md:127]:
```suggestion
discussione e il **[Team di Developers Italia](/it/contatti) interverrà per aiutare a completare
```

**Comment by sebbalex** [comment]
shomehow the callout is not centered, it seems depends from its size
`max-width: 60ch;` in `.callout` class

image

**Comment by bfabio** [comment]
@sebbalex It wasn't supposed to be centered. I removed the erroneous left margin.

**Comment by libremente** [comment]
Isn't it better in the center?

File: it/riuso/dichiarazione.md [hunk 9]
```diff
@@ -0,0 +1,177 @@
...
+see_also:
+  - label: Acquisizione di software
+    url: /it/riuso/acquisizione
+  - label: Pubblicazione di software
+    url: /it/riuso/pubblicazione
+---
+
+<!-- markdownlint-disable no-inline-html -->
+
+Secondo quanto previsto dalle [Linee guida su acquisizione e riuso di software per la PA](https://docs.italia.it/AgID/linee-guida-riuso-software/lg-acquisizione-e-riuso-software-per-pa-docs/it/bozza/attachments/allegato-d-guida-alla-modifica-di-software-open-source-preso-a-riuso-o-di-terzi.html)
```
Comment (libremente) [inline it/riuso/dichiarazione.md:17]:
```suggestion
Secondo quanto previsto dalle [Linee guida su acquisizione e riuso di software per la PA](https://docs.italia.it/italia/developers-italia/lg-acquisizione-e-riuso-software-per-pa-docs/it/stabile/attachments/allegato-d-guida-alla-modifica-di-software-open-source-preso-a-riuso-o-di-terzi.html)
```

**Review by libremente — CHANGES_REQUESTED** [review]

File: it/riuso/dichiarazione.md [hunk 10]
```diff
@@ -0,0 +1,177 @@
...
+Questa segnalazione permette all’Ente cedente di avere contezza di chi stia
+attivamente utilizzando un dato software e **facilitarne in questo modo
+l'interazione.**
+
+<div class="show-if-not-repo">
+  Le operazioni necessarie per effettuare la dichiarazione
+  variano in base all’esistenza o meno del software all’interno del catalogo.
+
+  <h2>
+    Se il software è presente nel catalogo di Developers Italia
```
Comment (libremente) [inline it/riuso/dichiarazione.md:34]:
```suggestion
    Se il software è presente nel catalogo
```

File: it/riuso/dichiarazione.md [hunk 11]
```diff
@@ -0,0 +1,177 @@
...
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="show-if-not-repo">
+  <h2>
+    Se il software non è presente nel catalogo di Developers Italia
```
Comment (libremente) [inline it/riuso/dichiarazione.md:118]:
```suggestion
    Se il software non è presente nel catalogo
```

**Review by libremente — CHANGES_REQUESTED** [review]

**Comment by bfabio** [comment]
> Isn't it better at the center?

I don't know, I find it less intrusive and less banner-y when not centered.

File: it/riuso/acquisizione.md [hunk 12]
```diff
@@ -8,6 +8,8 @@ subtitle:
 see_also:
   - label: Pubblicazione di software
     url: /it/riuso/pubblicazione
+  - label: Dichiarazione di presa a riuso di software
```
Comment (libremente) [inline it/riuso/acquisizione.md:11]:
```suggestion
  - label: Dichiarazione di riuso di software
```

File: it/riuso/dichiarazione.md [hunk 13]
```diff
@@ -0,0 +1,177 @@
+---
+title: Dichiarazione di presa a riuso di software
```
Comment (libremente) [inline it/riuso/dichiarazione.md:2]:
```suggestion
title: Dichiarazione di riuso di software
```

File: it/riuso/dichiarazione.md [hunk 14]
```diff
@@ -0,0 +1,177 @@
...
+<div class="row">
+  <div class="col-12 col-lg-6">
+    <div class="card-wrapper card-space">
+      <div class="card card-bg no-after">
+        <div class="card-body d-flex align-items-center">
+          <div class="categoryicon-top">
+            <span class="circled-step">1</span>
+          </div>
+          <p>
+            Visitare il <strong><a id="repo" rel="noreferrer noopener nofollow external" target="_blank" href="">repository relativo al software in questione</a></strong>
```
Comment (libremente) [inline it/riuso/dichiarazione.md:53]:
```suggestion
            Visitare il <strong><a id="repo" rel="noreferrer noopener nofollow external" target="_blank" href="">repository relativo al software in questione.</a></strong>
```

File: it/riuso/pubblicazione.md [hunk 15]
```diff
@@ -8,6 +8,8 @@ subtitle:
 see_also:
   - label: Acquisizione di software
     url: /it/riuso/acquisizione
+  - label: Dichiarazione di presa a riuso di software
```
Comment (libremente) [inline it/riuso/pubblicazione.md:11]:
```suggestion
  - label: Dichiarazione di riuso di software
```

**Review by libremente — CHANGES_REQUESTED** [review]
I don't particularly like "presa a riuso", let's try this way

**Review by libremente — APPROVED** [review]
:+1: I like it! Thanks @bfabio

**Comment by sebbalex** [comment]
🚀

**Pull request merged**